app/src/main/java/info/nightscout/androidaps/plugins/general/maintenance/MaintenancePlugin.kt

```diff
@@ -67,6 +67,10 @@ class MaintenancePlugin @Inject constructor(
     //todo replace this with a call on startup of the application, specifically to remove
     // unnecessary garbage from the log exports
     fun deleteLogs(keep: Int) {
+
```

Comment:
remove this if you want to merge into dev

Reply:
removed changed back